test/tstDetailsMutualReachabilityDistance.cpp

```diff
@@ -32,7 +32,10 @@ auto compute_core_distances(ExecutionSpace exec_space,
 
   ARBORX_ASSERT(points.extent_int(0) >= k);
   using MemorySpace = typename ExecutionSpace::memory_space;
-  ArborX::BVH<MemorySpace> bvh{exec_space, points};
+  ArborX::BasicBoundingVolumeHierarchy<
+      MemorySpace, ArborX::Details::PairIndexVolume<ArborX::Box>,
+      ArborX::Details::DefaultIndexableGetter, ArborX::Box>
```

Comment:
Aren't these the default template arguments?
If so, any particular reason to spell them out?

Reply:
`MaxDistance` functor has been converted to the new `(Predicate, Value)` type. Calling it through `BVH<MemorySpace>` would result in an error.